README.md

```diff
@@ -100,30 +162,41 @@ This will take approximately a minute to run, with the majority of the time spen
-getServicesResult = await enclaveCtx.getServices()
-services = getServicesResult.value
 ```
+========================================= Kurtosis Modules =========================================
+GUID                                             LocalPortBindings
+ethereum-kurtosis-module.1649105153-1649105156   1111/tcp -> 127.0.0.1:55897
 
-...we see three service IDs:
+========================================== User Services ==========================================
+GUID                         ID                LocalPortBindings
+bootnode-1649105160          bootnode          30303/tcp -> 127.0.0.1:55902
+                                               30303/udp -> 127.0.0.1:49555
+                                               8545/tcp -> 127.0.0.1:55903
+                                               8546/tcp -> 127.0.0.1:55901
+ethereum-node-1-1649105164   ethereum-node-1   8545/tcp -> 127.0.0.1:55907
+                                               8546/tcp -> 127.0.0.1:55908
+                                               30303/tcp -> 127.0.0.1:55909
+                                               30303/udp -> 127.0.0.1:56275
+ethereum-node-2-1649105167   ethereum-node-2   30303/udp -> 127.0.0.1:51099
+                                               8545/tcp -> 127.0.0.1:55964
+                                               8546/tcp -> 127.0.0.1:55965
+                                               30303/tcp -> 127.0.0.1:55963
 
-```javascript
-Set(3) { 'bootnode', 'ethereum-node-1', 'ethereum-node-2' }
 ```
 
 But what just happened?
 
-Starting networks is a very common task in Kurtosis, so we provide [a framework called "modules"](https://docs.kurtosistech.com/modules.html) for making it dead simple. An executable module is basically a chunk of code that responds to an "execute" command, packaged as a Docker image, that runs inside a Kurtosis enclave - sort of like Docker Compose on steroids. In the steps above, we called `enclaveCtx.loadModule` to load [the Ethereum module](https://github.com/kurtosis-tech/ethereum-kurtosis-module) into the enclave with module ID `eth-module`, and `ethModuleCtx.execute` to run it. The Ethereum module doesn't take any parameters at load or execute time (hence the `{}`), but other modules do.
+Starting networks is a very common task in Kurtosis, so we provide [a framework called "modules"](https://docs.kurtosistech.com/modules.html) for making it dead simple. An executable module is basically a chunk of code that responds to an "execute" command, packaged as a Docker image, that runs inside a Kurtosis enclave - sort of like Docker Compose on steroids. In the steps above, we executed `kurtosis module exec` to load [the Ethereum module](https://github.com/kurtosis-tech/ethereum-kurtosis-module) into a new enclave. The Ethereum module doesn't take any parameters at load or execute time, but other modules do so, you can add them using the `--execute-params` flag (e.g: --execute-params '{"numDatastores":2}')
```

Comment:
s/into a new enclave/into the `demo` enclave/


also recommend swapping..
```
but other modules do so, you can add them using the `--execute-params` flag (e.g: --execute-params '{"numDatastores":2}')
```
..for...
```
but other modules do via the `--execute-params` flag.
```
so we don't add too much extraneous information to the user.